xml/System.Diagnostics/DiagnosticListener.xml

```diff
@@ -439,10 +439,10 @@ If `isEnabled` is `null`, no filtering is done (all overloads of <xref:System.Di
         <Parameter Name="onActivityExport" Type="System.Action&lt;System.Diagnostics.Activity,System.Object&gt;" Index="3" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="observer">To be added.</param>
-        <param name="isEnabled">To be added.</param>
-        <param name="onActivityImport">To be added.</param>
-        <param name="onActivityExport">To be added.</param>
+        <param name="observer">A subscriber.</param>
+        <param name="isEnabled">A delegate that filters events based on their name and up to two context objects (which can be null), or null to if an event filter is not desirable.</param>
```

Comment:
Is this talking about a C# null? If yes, then it should it be `<see langword="null" />`. If not (like with Json Nulls), then ignore my suggestion:

```suggestion
        <param name="isEnabled">A delegate that filters events based on their name and up to two context objects (which can be <see langword="null" />), or <see langword="null" />to if an event filter is not desirable.</param>
```